packages/mds-ingest-service/service/telemetry_validators.ts

```diff
@@ -112,3 +114,17 @@ export const { validate: validateTelemetryAnnotationDomainCreateModel, $schema:
 
 export const { validate: validateTelemetryAnnotationDomainModel, $schema: TelemetryAnnotationSchema } =
   SchemaValidator<TelemetryAnnotationDomainModel>(telemetryAnnotationDomainModelSchema(), { useDefaults: true })
+
+export const getH3BinsOptionsSchema: Schema<GetH3BinOptions> = <const>{
+  type: 'object',
+  properties: {
+    k: { type: 'number', minimum: 1 },
```

Comment:
May be worth specifying that this should be an `integer` versus just a number:
```suggestion
    k: { type: 'integer', minimum: 1 },
```